DigitalLearningSolutions.Data.Tests/DataServices/CourseDataServiceTests.cs

```diff
@@ -933,5 +933,30 @@ public void CreateNewCentreCourse_correctly_adds_new_course()
                 transaction.Dispose();
             }
         }
+
+        [Test]
+        public void GetDelegateAnswersForCourseAdminFields_returns_expected_results()

```

Comment:
I'm not sure the test name quite matches up with the assertions.
Although we check the number of results that has come back, we only check for equality on one of the results.
Given the list of delegate id's will be a constant can we compare those too?